### ISSUE TITLE
works in BLorient743

### ISSUE THREAD

**Issue by thea-m** [BetaMasaheft/Documentation#2107]
MS BLorient743 (not digitised, text transcribed in the record on the branch DR_22_04_BLorient743) contains several works I could not identify 
https://archive.org/details/catalogueofethio00brituoft/page/208/mode/2up?view=theater

I would consider creating a new record for Wright's 1abc (or 1a and 1b separetedly?) and for his 2 (Commentary on the Song of Songs). The descriptions of commentaries he lists under 3 and 4 are maybe too vague to create records on their basis. I would also not create records for the various lists of works in foreign languages that follow. 
But what do you think?
@DenisNosnitsin1970 @eu-genia

**Comment by thea-m** [comment]
Any ideas? :)

**Comment by eu-genia** [comment]
I wonder if lists such as 6-7-8 deserve a NAR or something like that, they might interest someone to look for them

**Comment by DenisNosnitsin1970** [comment]
I see at the end of the manuscript Enbaqom is mentioned. If he is ecchege Enbaqom of Dabra Libanos, there may be some more hints in the EAE article about him.

**Comment by thea-m** [comment]
> I see at the end of the manuscript Enbaqom is mentioned. If he is ecchege Enbaqom of Dabra Libanos, there may be some more hints in the EAE article about him.

No, I had checked it, nothing helpful. It is not clear whether anyone ever actually saw the manuscript.

**Comment by DenisNosnitsin1970** [comment]
I assume that Armenian-related pieces were attended by some people who worked on Ethio-Armenian relations, 
so some hints may be in EAE "Armenia", or elsewhere in articles about this subject. There is a small "genre" of (thematic) list of words from "remote" languages with Ge'ez or Amharic translations, worth of ID of whatever type.  Also I noticed smth about Or 743 here, https://www.jstor.org/stable/10.1163/j.ctv1sr6k2b.16?seq=1.

**Comment by thea-m** [comment]
Thank you!
